src/ripple/net/impl/DatabaseBody.ipp

```diff
@@ -17,6 +17,8 @@
 */
 //==============================================================================
 
+#include <ripple/core/SQLInterface_global.h>
```

Comment:
Not in this changeset I know, buy why is it important that these functions are inlined? I'd expect these functions (except the templates) to live in a cpp file.

Reply:
Not in this changeset, I agree.